test/config/va.json

```diff
@@ -26,7 +26,11 @@
       ]
     },
     "features": {
-    }
+      "SimplifiedVAHTTP": true
```

Comment:
I think rather than adding this feature flag to the `config` tree it should be deleted from `config-next` and the `features` package. It's been deployed everywhere and the legacy VA code was deleted in https://github.com/letsencrypt/boulder/commit/9f5c1b9e25e529bed055fd6e063f44be9e2fa436. It doesn't have any effect on Boulder's runtime behaviour.